api-inference-community/docker_images/flair/app/main.py

```diff
@@ -1,5 +1,6 @@
 import logging
 import os
+import functools
```

Comment:
Please fix style, this is not sorted properly 

(I see that the automatic check is not checking this directory. I'll send a PR to fix)